test/integration/style-spec/tests/bad-color.output.json

```diff
@@ -10,5 +10,17 @@
   {
     "message": "layers[1].paint.fill-outline-color: color expected, array found",
     "line": 26
+  },
+  {
+    "message": "layers[2].paint.fill-color: color expected, \"__proto__\" found",
```

Comment:
Should the message beyter describe the problem? Is there a word missing after the "found"?

Reply:
Ignore this, my mistake...